tools/devtool

```diff
@@ -293,6 +302,7 @@ run_devctr() {
         --volume "$FC_ROOT_DIR:$CTR_FC_ROOT_DIR" \
         --env OPT_LOCAL_IMAGES_PATH="$(dirname "$CTR_MICROVM_IMAGES_DIR")" \
         --env PYTHONDONTWRITEBYTECODE=1 \
+        ${docker_proxies[@]} \
```

Comment:
I think it would be a good idea to use the `docker_args` array, since it's already there for constructing a dynamic list of docker arguments.

Reply:
I firstly try to add this flag into `docker_args`, but I get error below:
```
unknown flag: --env http_proxy
See 'docker run --help'.
```
maybe it's because the master code use `"${docker_args[@]}"` instead of `${docker_args[@]}`.
Also I can try to use `docker_args` but I will change `"${docker_args[@]}"` into `${docker_args[@]}` in the new commit